pkgs/development/python-modules/myst-parser/default.nix

```diff
@@ -28,7 +28,7 @@ buildPythonPackage rec {
     owner = "executablebooks";
     repo = pname;
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-wgRZwafAF05LtwjH6SVKzsY7bKeZ6lUlM3dB5PdOn1E=";
+    sha256 = "sha256-5l22iEteXNcgyW8Qq7MTZeHfN8CvledBPq7dZiytPkw=";
   };
 
   postPatch = ''
```

Comment:
Would you mind removing these `substituteInPlace`? since it's no longer needed

Reply:
Done